.github/workflows/main.yml

```diff
@@ -27,7 +27,7 @@ env:
   OUTFILE_LOWPU_MU: "mw_lowPU_mu.hdf5"
   OUTFILE_LOWPU_EE: "mz_lowPU_ee.hdf5"
   OUTFILE_LOWPU_MUMU: "mz_lowPU_mumu.hdf5"
-  DATAPATH: "/scratch/shared/NanoAOD/"
```

Comment:
This changes the processed files, I assume because you want to use v6. But Then a comparison of the CI is more difficult, so I suggest we either move the v6 samples to scratch or change to eos in a separate PR.

Reply:
We already validated the change to V6 in the PR which updated the production tags. (By that time temporarily switching the paths)

Reply:
That was in https://github.com/WMass/WRemnants/pull/444